packages/apollo-angular/http/src/utils.ts

```diff
@@ -143,7 +143,7 @@ export function prioritize<T>(...values: T[]): T {
   return picked;
 }
 
-export function createHeadersWithClientAwereness(context: Record<string, any>) {
+export function createHeadersWithClientAwareness(context: Record<string, any>) {
```

Comment:
This function is publicly available via `import {createHeadersWithClientAwereness} from 'apollo-angular/http/utils';`. I assumed this was an accident and it was not mean to be public. So we can rename it without considering as a breaking change.

If it was actually meant to be public we can re-introduce the old spelling and mark it as deprecated. WDYT ?